Lib/test/test_fstring.py

```diff
@@ -1049,6 +1064,36 @@ def test_backslash_char(self):
         self.assertEqual(eval('f"\\\n"'), '')
         self.assertEqual(eval('f"\\\r"'), '')
 
+    def test_debug_conversion(self):
```

Comment:
Please add also tests for expressions containing "=": with operators "==", "!=", "<=", ">=", ":=" and string literals containing "=" and for format spec containing "=" (`f'{x:=10}'` where `x` is a number).

Reply:
Good idea.

Reply:
I've added these tests.